src/main/java/net/io/kino/service/impl/ReservationServiceImpl.java

```diff
@@ -0,0 +1,63 @@
+package net.io.kino.service.impl;
+
+import net.io.kino.model.*;
+import net.io.kino.repository.OrdersRepository;
+import net.io.kino.repository.TicketTypesRepository;
+import net.io.kino.service.ReservationService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+import java.util.Optional;
+
+public class ReservationServiceImpl implements ReservationService {
+
+    @Autowired
+    private OrdersRepository orders;
+
+    @Autowired
+    private TicketTypesRepository ticketTypes;
+
+    @Override
+    public boolean createOrder(List<Ticket> tickets, PersonalDetails client) {
```

Comment:
createOrder wymaga przekazania ticketów, żeby stworzyć Order
A żeby stworzyć Ticket to trzeba mu przekazać Order.
Coś nie teges :')